src/mixins/filing-template-mixin.ts

```diff
@@ -272,7 +272,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           name: FilingTypes.DISSOLUTION,
           certifiedBy: this.getCertifyState.certifiedBy,
           date: this.getCurrentDate,
-          isFutureEffective: this.getEffectiveDateTime.isFutureEffective
+          isFutureEffective: null
```

Comment:
The filing now supports 3 states:
- null: no option selected
- true: user selected future effective
- false: user selected immediate